pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = [
+    "setuptools>=59.6",
+    "pybind11>=2.10.0",
+    "cmake>=3.22",
+    "numpy>=1.21", 
+    "wheel",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "diskannpy"
+version = "0.4.0"
+
+description = "DiskANN Python extension module"
+# readme = "../README.md"
+requires-python = ">=3.7"
+license = {text = "MIT License"}
+dependencies = [
+    "numpy"
+]
+authors = [
+    {name = "Harsha Vardhan Simhadri", email = "harshasi@microsoft.com"},
+    {name = "Dax Pryce", email = "daxpryce@microsoft.com"}
```

Comment:
Shikhar Jaiswal contributed too.

Reply:
I intended this author list to actually be the list of current maintainers, not all contributors: https://peps.python.org/pep-0621/#authors-maintainers

Reply:
(that said, we should consider adding https://github.com/all-contributors/app to our project and using it for everyone who has made contributions - of all types, not just code contributions!)